lib/org/chromium/webidl/LogLevel.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.ENUM({
+  package: 'org.chromium.webidl',
+  name: 'LogLevel',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'shortName',
+    },
+    {
+      class: 'String',
+      name: 'consoleMethodName',
+    },
+  ],
+
+  values: [
+    {
+      name: 'DEBUG',
+      shortName: 'DEBG',
+      label: 'Debug',
+      consoleMethodName: 'debug',
+    },
+    {
+      name: 'LOG',
```

Comment:
Yeah. That's true. I'm following the Javascript `console` interface method names. I suppose I could just drop this one and export `log` as a log-at-info-level. Does that seem more reasonable?